core/src/it/scala/org/ensime/core/RichPresentationCompilerSpec.scala

```diff
@@ -17,10 +17,12 @@ import org.ensime.api._
 import org.ensime.fixture._
 import org.ensime.indexer._
 import org.ensime.indexer.SearchServiceTestUtils._
+import org.ensime.intg.ExcludeInTravis
 import org.ensime.util.EnsimeSpec
 import org.ensime.util.file._
 import org.ensime.vfs.EnsimeVFS
 
+@ExcludeInTravis
```

Comment:
yes that's right, but that's ok. The slow tests tend to be pretty big, and we might be able to run just one of them, e.g. in reverse lookups.